extensions/amp-analytics/0.1/resource-timing.js

```diff
@@ -246,13 +246,18 @@ function serialize(entries, resourceTimingSpec, win) {
  * Serializes resource timing entries according to the resource timing spec.
  * @param {!JsonObject} resourceTimingSpec
  * @param {!Window} win
+ * @param {number} responseAfter A timestamp (relative to navigationStart) for
+ *   filtering resource timing entries. Only entries that have fully downloaded
+ *   after this time will be included.
  * @return {!Promise<string>}
  */
-export function serializeResourceTiming(resourceTimingSpec, win) {
+export function serializeResourceTiming(
```

Comment:
Nit: Please have win be the first param.

Reply:
Done